.gitattributes

```diff
@@ -1,2 +1,5 @@
 # Set the default behavior, in case contributors don't have core.autocrlf set.
 * text=auto
+
+# prevent Github from including yarn source
+.yarn/releases/*.cjs linguist-vendored
```

Comment:
The file added is `.yarn/releases/yarn-1.22.11.js` but this is referencing `.yarn/releases/*.cjs`.

Reply:
I went based on your suggestion. I'll edit to add both. As a heads up, I'm using the same and seeing the same. Probably best to support both.
![image](https://user-images.githubusercontent.com/13071055/131720663-64f69cf7-8202-47e7-bcec-ff9b6294e54d.png)